examples/1-node-example.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 The Cockroach Authors
```

Comment:
What is the purpose of this new example?

1. You will never run just one node
2. You cannot run a CR database without a PV

Reply:
As explained in [this earlier comment](https://github.com/cockroachdb/cockroach-operator/pull/142#discussion_r475839332): It's not a new example. It is a renamed example that was already in the repo. I'll delete from this PR.